cli/src/formatter.rs

```diff
@@ -389,9 +410,16 @@ fn color_for_name(color_name: &str) -> Result<Color, config::ConfigError> {
         "bright magenta" => Ok(Color::Magenta),
         "bright cyan" => Ok(Color::Cyan),
         "bright white" => Ok(Color::White),
-        _ => Err(config::ConfigError::Message(format!(
-            "invalid color: {color_name}"
-        ))),
+        _ => {
+            if color_name.starts_with("rgb(") && color_name.ends_with(")") {
```

Comment:
Feature aside, it's probably easier and less ambiguous to parse `#rrggbb` syntax.

Reply:
Thanks for the feedback. Not sure about ambiguity, but it's certainly easier to parse `#rrggbb`. I've changed it. TBH I opened the PR prematurely and was pleasantly surprised to get feedback so quickly. Should be good to look at now though.